### DISCUSSION TITLE
Using 'M' as default for selecting inner brackets

### DISCUSSION THREAD

**Discussion by ff2400t** [helix-editor/helix#2266] [Ideas] [answered]
I would like to suggest using 'M' to default to selecting inner text object, this would make changing and deleting text inside these objects easier . This is a very common use and i think having this as default would be great. Another Reason for having this as default would be that in vim the same thing requires fewer key strokes `ciw` while in helix this would be `miwc`.

**Comment by the-mikedavis** [comment, marked as answer]
You can bind this in your `config.toml`:

```toml
[keys.normal]
"M" = "select_textobject_inner"
```

Golf-wise, many consider <kbd>Shift</kbd> to count as a keystroke so `M` isn't really better than `mi`. Plus part of the vision is to prize consistency and ease of memory over brevity (see [here](https://github.com/helix-editor/helix/blob/b65fb0f64ad6cf7a54f4dda92bebfc22d80b236f/docs/vision.md?plain=1#L19)). Whether you use `mi` or `ma` or `ms` more depends on how you end up using Helix so choosing one to get a special keybind is contentious.
